tests/test_data_schema.py

```diff
@@ -153,7 +153,41 @@ def testConstructQueryRustworkX(self):
         with self.subTest(msg="11. field doesn't exist"):
             with self.assertRaises(ValueError):
                 SCHEMA._Schema__construct_query_rustworkx(input_ids={'assembly_id': "1", "interface_id": "1", "entry_id": "4HHB"}, input_type="interface", return_data_list=["aaa"])
-        
+        with self.subTest(msg="12. two arguments (polymer_entity_instances)"):
+            query = SCHEMA._Schema__construct_query_rustworkx(input_ids={'instance_ids': ["4HHB.A", "4HHB.C"]}, input_type="polymer_entity_instances", return_data_list=["rcsb_polymer_instance_annotation"])
+            response_json = requests.post(headers={"Content-Type": "application/graphql"}, data=query, url=pdb_url).json()
+            self.assertNotIn("errors", response_json.keys())
+        with self.subTest(msg="13. entries id_list provided"):
+            query = SCHEMA._Schema__construct_query_rustworkx(input_type="entries", return_data_list=["exptl"], input_ids=["4HHB", "1IYE"])
+            response_json = requests.post(headers={"Content-Type": "application/graphql"}, data=query, url=pdb_url).json()
+            self.assertNotIn("errors", response_json.keys())
+        with self.subTest(msg="14. id_list provided with incorrect input_type"):
```

Comment:
Is there a specific reason that this test is placed here? Otherwise, I was thinking it would be good to have at the end with the other ValueError and reflecting the structure of the previous test